tools/Cargo.toml

```diff
@@ -56,6 +56,7 @@ reqwest = "0.11.3"
 rls-analysis = "0.18.1"
 rls-data = "0.19.1"
 scip = "0.3.3"
+serde_json = { version = "1.0.113", features = ["preserve_order", "std"] }
```

Comment:
what about the copy of this [up on line 16](https://github.com/mozsearch/mozsearch/blob/c2a2e4a6476a4fd278953876072631eac3cc23e4/tools/Cargo.toml#L16)  that doesn't include "std" in the features?